### PR TITLE
Added Elasticsearch Curator chart

### PR DESCRIPTION
**Pull request by tmestdagh**
This PR adds the elasticsearch curator Helm chart.
After installing this chart, a CronJob will be created which runs the curator according to a schedule and configuration you can set.

### CODE REVIEW

File: incubator/elasticsearch-curator/README.md [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+This chart will do the following:
+
+* Create a CronJob which runs the Curator
+
+## Installing the Chart
+
+To install the chart, use the following:
+
+```console
+$ helm install stable/elasticsearch-curator
```
Comment (gianrubio) [inline incubator/elasticsearch-curator/README.md]: should be incubator

File: incubator/elasticsearch-curator/templates/configmap.yaml [hunk 2]
```diff
@@ -0,0 +1,15 @@
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: curator-config
```
Comment (gianrubio) [inline incubator/elasticsearch-curator/templates/configmap.yaml:4]: you missed labels here

File: incubator/elasticsearch-curator/templates/NOTES.txt [hunk 3]
```diff
@@ -0,0 +1,6 @@
+A CronJob will run with schedule {{ .Values.cronjob.schedule }}.
+
+The Jobs will not be removed automagically when deleting this Helm chart.
+To remove these jobs, run the following :
+
+    kubectl -n logging delete job -l app=elasticsearch-curator
```
Comment (gianrubio) [inline incubator/elasticsearch-curator/templates/NOTES.txt]: don't hardcode labels here, use app={{ template "elasticsearch-curator.name" . }}

File: incubator/elasticsearch-curator/templates/cronjob.yaml [hunk 4]
```diff
@@ -0,0 +1,48 @@
+apiVersion: batch/v1beta1
```
Comment (gianrubio) [inline incubator/elasticsearch-curator/templates/cronjob.yaml:1]: don't hardcode the apiVersion here, see these example [helper](https://github.com/kubernetes/charts/blob/master/incubator/jaeger/templates/_helpers.tpl#L3-L12) and [how to apply it](https://github.com/kubernetes/charts/blob/ae2a474b0c832edc92345bd87e48130c50ced50c/incubator/jaeger/templates/spark-cronjob.yaml#L2-L3)

**Review by gianrubio — CHANGES_REQUESTED** [review]
just minor changes

File: incubator/elasticsearch-curator/values.yaml [hunk 5]
```diff
@@ -0,0 +1,83 @@
...
+  #     timeout: 30
+  #     master_only: False
+  #   logging:
+  #     loglevel: INFO
+  #     logfile:
+  #     logformat: default
+  #     blacklist: ['elasticsearch', 'urllib3']
+
+
+resources: {}
```
Comment (timstoop) [inline incubator/elasticsearch-curator/values.yaml:67]: This one is not in the table on your README.

File: incubator/elasticsearch-curator/values.yaml [hunk 6]
```diff
@@ -0,0 +1,83 @@
...
+  # resources, such as Minikube. If you do want to specify resources, uncomment the following
+  # lines, adjust them as necessary, and remove the curly braces after 'resources:'.
+  # limits:
+  #  cpu: 100m
+  #  memory: 128Mi
+  # requests:
+  #  cpu: 100m
+  #  memory: 128Mi
+
+nodeSelector: {}
```
Comment (timstoop) [inline incubator/elasticsearch-curator/values.yaml]: This one and the next two are also not in you README.

**Review by timstoop — CHANGES_REQUESTED** [review]

File: incubator/elasticsearch-curator/OWNERS [hunk 7]
```diff
@@ -0,0 +1,6 @@
+approvers:
+  - tmestdagh
+  - gianrubio
```
Comment (unguiculus) [inline incubator/elasticsearch-curator/OWNERS:3]: @gianrubio I guess you should be added as maintainer as well.

File: incubator/elasticsearch-curator/templates/cronjob.yaml [hunk 8]
```diff
@@ -0,0 +1,49 @@
+{{- if or (.Capabilities.APIVersions.Has "batch/v1beta1") (.Capabilities.APIVersions.Has "batch/v2alpha1") -}}
```
Comment (unguiculus) [inline incubator/elasticsearch-curator/templates/cronjob.yaml]: We shouldn't have to support `batch/v2alpha1` any more.

File: incubator/elasticsearch-curator/templates/cronjob.yaml [hunk 9]
```diff
@@ -0,0 +1,49 @@
...
+            - name: config-volume
+              configMap: 
+                name: curator-config
+          restartPolicy: Never
+          containers:
+            - name: {{ .Chart.Name }}
+              image: "{{ .Values.image.repository }}:{{ .Values.image.tag }}"
+              imagePullPolicy: {{ .Values.image.pullPolicy }}
+              volumeMounts:
+              - name: config-volume
```
Comment (unguiculus) [inline incubator/elasticsearch-curator/templates/cronjob.yaml]: Make list indentation consistent. This should indent one more level.

**Review by unguiculus — CHANGES_REQUESTED** [review]

File: incubator/elasticsearch-curator/templates/_helpers.tpl [hunk 10]
```diff
@@ -0,0 +1,36 @@
+{{/* vim: set filetype=mustache: */}}
+{{/*
+Expand the name of the chart.
+*/}}
+{{/*
```
Comment (unguiculus) [inline incubator/elasticsearch-curator/templates/_helpers.tpl]: Remove obsolete comment.

File: incubator/elasticsearch-curator/templates/_helpers.tpl [hunk 11]
```diff
@@ -0,0 +1,36 @@
+{{/* vim: set filetype=mustache: */}}
+{{/*
```
Comment (unguiculus) [inline incubator/elasticsearch-curator/templates/_helpers.tpl]: This comment belongs to the name partial below.

**Review by unguiculus — CHANGES_REQUESTED** [review]

**Comment by tmestdagh** [comment]
PullRequest got merged!  Thanks for the reviews @gianrubio @timstoop @unguiculus

**Pull request merged**